modules/EmailTemplates/EmailTemplate.php

```diff
@@ -814,13 +620,20 @@ public function parse_template($string, &$bean_arr)
             if ($bean_name == 'Leads' || $bean_name == 'Prospects') {
                 $bean_name = 'Contacts';
             }
-
-            if (isset($this) && isset($this->module_dir) && $this->module_dir == 'EmailTemplates') {
-                $string = $this->parse_template_bean($string, $bean_name, $focus);
-            } else {
-                $string = EmailTemplate::parse_template_bean($string, $bean_name, $focus);
+            $string = self::parse_template_bean($string, $focus->module_name, $focus);
+
+            foreach ($focus->field_defs as $focus_name => $focus_arr) {
+                if ($focus_arr['type'] == 'relate') {
+                    if (isset($focus_arr['module']) && $focus_arr['module'] != '' && $focus_arr['module'] != 'EmailAddress') {
+                        $idName = $focus_arr['id_name'];
+                        $relate_focus = BeanFactory::getBean($focus_arr['module'], $focus->$idName);
+                        if(!$relate_focus) continue;
+                        $string = self::parse_template_bean($string, $focus_arr['name'], $relate_focus);
+                    }
+                }
```

Comment:
I think this PR will need to be rebased as I get conflicts with this change and what is already on hotfix-7.10.x:

https://github.com/salesagility/SuiteCRM/blob/hotfix-7.10.x/modules/EmailTemplates/EmailTemplate.php#L803